### ISSUE TITLE
JWT Populate Lambda - overwrite birthday ends up in array

### ISSUE THREAD

**Issue by jerryhopper** [FusionAuth/fusionauth-issues#398]
### Description 
When overwriting the birthdate field with a lambda, the original value doesnt get replaced, instead the birthdate field becomes and array with the old, and new value

### Steps to reproduce
1. In fustionauth, create lambda that overwrites the birthdate
2. Login fusionauth & examine decode the token.
3. See the problem

### Expected behavior
The birthdate string should be replaced.

### Example Lamda & result

```
function populate(jwt, user, registration) {
   jwt.birthdate = "overwritten";
}

```

```
birthdate: {
  'overwritten',
  '1973-06-15'
}
```

### Platform
 - latest fusionauth

**Comment by jerryhopper** [comment]
Additional info from chat with robotdan:

> OIDC does define birthdate as a string, but because it is not an OAuth2 reserved claim we are treating it generically which causes us to merge the value.

**Comment by robotdan** [comment]
When merging the values from a lambda, we should probably just always set and not merge when the claim is not reserved. 

In theory we could enforcer the OIDC spec which says some claims a strings, etc  and then not allow the lambda to set them as an array, but that seems a bit heavy. Once you're using a lambda you're into power user territory anyway.

**Comment by robotdan** [comment]
When I initially looked at this, I thought I understood how this would happen, but I have been unable to recreate your symptom. 

Do you have any other details that may help me narrow this down? 

We don't add the `birthdate` claim to the JWT outside of a Lambda, so I'm not sure where the `1973-06-15` value is coming from in your example.

**Comment by jerryhopper** [comment]
Excuse me for the late reply.  i will create a example for you.

**Comment by robotdan** [comment]
Thanks!! I don't know yet how to recreate, but I think I may have an idea of how it could happen (and it will be fixed in the upcoming release).

But if you do know how to recreate, I'd still take an example to prove it has been fixed.

**Comment by robotdan** [comment]
FYI, while I was unable to recreate, this may be resolved by a change I made in 1.14.0.

If you are unable to recreate on 1.14.0 we can close out the issue.

**Comment by robotdan** [comment]
Closing, let me know if you can reproduce and we can go from there.
